src/DrevOps/Sniffs/TestingPractices/DataProviderMatchesTestNameSniff.php

```
@@ -0,0 +1,296 @@
+   *   The file being scanned.
+   * @param int $functionPtr
+   *   The position of the function token.
+   *
+   * @return string|null
+   *   The provider method name, or NULL if not found or external.
+   */
+  private function findDataProviderAnnotation(File $phpcsFile, int $functionPtr): ?string {
+    $tokens = $phpcsFile->getTokens();
+
+    // Search backward for docblock before function.
+    $comment_end = $phpcsFile->findPrevious(T_DOC_COMMENT_CLOSE_TAG, $functionPtr - 1);
```

PHPMD (2.15.0)

The variable $comment_end is not named in camelCase. (undefined)

(CamelCaseVariableName)